### ISSUE TITLE
How is the fine tuning of real data done?

### ISSUE THREAD

**Issue by sfxz035** [Algolzw/EBSR#13]
Hi, I have some questions about the fine tuning strategy.  In table 1, fine-tuning the real data get a higher PSNR. I did this and it turned out worse, so i hope to get your answer for some questions. What is the specific strategy of fine tuning? What is the initial learning rate? Is there a learning rate decay？Whether to control certain weights to disable learning? Or is there some other strategy to focus on？

**Comment by Algolzw** [comment]
Finetuning on the real dataset should always improve the performance. One thing you need to notice is that the training pipeline of the real data is a little different from the synthetic data.
1. You need to use AlignedL1 loss for the SR image and GT image, which you can find in [here](https://github.com/Algolzw/EBSR/blob/1abeb3e6193882e7de7d60f5e25f7481df1cc3a6/utils/metrics.py#L130).
2. You need to use AlignedPSNR to calculate the PSNR value, see [here](https://github.com/Algolzw/EBSR/blob/1abeb3e6193882e7de7d60f5e25f7481df1cc3a6/utils/metrics.py#L237).
3. You may need to change the patch_size to 48 or 64.
4. All weights are finetuned.
5. And for specific options, we use "--n_GPUs 8 --lr 0.00006 --decay 40-80 --batch_size 8 --patch_size 64"

**Comment by sfxz035** [comment]
Got it. That helps a lot. Thank you.
